zerver/webhooks/slack/view.py

```diff
@@ -52,8 +54,12 @@ def convert_mentions(text: str) -> str:
         )
 
         if slack_usermention_match:
-            # TODO: Callback to Slack API
-            pass
+            # We convert user mentions into a mention-like message to because
+            # there are no way of mapping between a users Slack and Zulip account
+
```

Comment:
I'd revise to be not have the empty line after the comment. Also, the comment could be simplified...

`# Convert Slack user mentions to a mention-like syntax since there is no way to map Slack and Zulip users.`